File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/account/BankType.java
package com.example.cf_sdk.logic.changebanksdk.model.account;

import com.google.gson.annotations.SerializedName;

/**
 *
 * Bank types.
 */

public enum BankType {
    @SerializedName("CARDHOLDER_BANK")
    CARDHOLDER_BANK,

    @SerializedName("ACH_BANK")
    ACH_BANK,
}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/account/BankType.kt
package com.example.cf_sdk.changebankapi.model.account

import com.google.gson.annotations.SerializedName

/**
 *
 * Bank types.
 */
enum class BankType {
    @SerializedName("CARDHOLDER_BANK")
    CARDHOLDER_BANK, @SerializedName("ACH_BANK")
    ACH_BANK
}